### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by buff0000n** [buff0000n/shawzintab#1]
This an open issue for people to leave feedback

**Comment by Xeno-Cortex** [comment]
Well, i don't know if this is the right location or format to do this in since it's pretty barren as it is, but i was futzing around and noticed that the icons for the PS4 buttons were backwards  compared to the way the music was played. It adds a new layer of difficulty in trying to learn a new song.

**Comment by buff0000n** [comment]
> Well, i don't know if this is the right location or format to do this in since it's pretty barren as it is, but i was futzing around and noticed that the icons for the PS4 buttons were backwards compared to the way the music was played. It adds a new layer of difficulty in trying to learn a new song.

The first, lowest string is on the bottom.  Is that not Square by default on the PS4?

**Comment by Xeno-Cortex** [comment]
On ps4, the lowest string is square, then X, then circle- and the frets go from L1, to R1, to R2, in left-to-right order. I even reset my controls to default to make sure that was correct. I'm glad to clear up any confusion.

Edit: i think the problem comes from how it looks like it should be played from right-to-left in the app, but the music plays correctly and sensibly from left-to-right.

**Comment by buff0000n** [comment]
> Edit: i think the problem comes from how it looks like it should be played from right-to-left in the app, but the music plays correctly and sensibly from left-to-right.

Yeah it's left to right.  The app is the in-game note crawl turned 90 degrees counterclockwise, with the string labels moved to the left to match how guitar tablature works.
